client/landing/stepper/declarative-flow/newsletter.ts

```diff
@@ -45,7 +45,13 @@ export const newsletter: Flow = {
 					);
 
 				case 'newsletterSetup':
-					return window.location.replace( '/start/newsletter/domains' );
+					window.location.replace(
```

Comment:
Nit: `return` is missing (while other places seem to have `return` in place). Harmless because there are no more codes to execute after this switch block.

Reply:
Ahh good catch. I did not mean to leave this out. I will put the return back in. Give me a sec.

Reply:
My bad, you already fixed that too. 🙇  BTW I noticed one issue with your refactor after conflicts merge, I just fixed it [here](https://github.com/Automattic/wp-calypso/pull/66861/commits/c6c33ae7bdc141c3a4099a0428efbc52fd6151dc).